File: controledenotas.py
n_alunos = int(input("Quantos alunos existem na turma? "))
soma_notas = 0
aprovados = 0
recuperacao = 0
reprovados = 0

for i in range(n_alunos):
    nota = float(input(f"Nota do aluno {i+1}: "))
    soma_notas += nota
    
    if nota >= 7:
        aprovados += 1
    elif nota >= 5:
        recuperacao += 1
    else:
        reprovados += 1

print("-" * 20)
print(f"Média da turma: {soma_notas / n_alunos:.2f}")
print(f"Aprovados: {aprovados}")
print(f"Recuperação: {recuperacao}")
print(f"Reprovados: {reprovados}")

File: controledenotas.c
#include <stdio.h>

int main() {
    int alunos, i;
    float nota, soma = 0, media;
    int aprovados = 0, recuperacao = 0, reprovados = 0;

    printf("Quantos alunos tem na turma? ");
    scanf("%d", &alunos);

    for (i = 0; i < alunos; i++) {
        printf("Nota do aluno %d: ", i + 1);
        scanf("%f", &nota);
        soma += nota;

        if (nota >= 7.0) aprovados++;
        else if (nota >= 5.0) recuperacao++;
        else reprovados++;
    }

    media = soma / alunos;

    printf("\n=== RESULTADO DA TURMA ===\n");
    printf("Media da turma: %.2f\n", media);
    printf("Aprovados (>=7): %d\n", aprovados);
    printf("Recuperacao (5 a 6.9): %d\n", recuperacao);
    printf("Reprovados (<5): %d\n", reprovados);

    return 0;
}